docs/configuration.asciidoc

```diff
@@ -36,20 +36,22 @@ With this you can use any configuration source that you configured on the `IConf
 
 NOTE: By simply calling `app.UseElasticApm()` without the overload, the agent will read configurations only from environment variables.
 
+NOTE: For better user experience, `LogLevel` for internal apm logger is used from standard `Logging` section inside `appsettings.json` with `Apm` category name.
+
 This is a typical `appsettings.json` file that contains some sample configuration. The part below `ElasticApm` is fetched by the agent if the corresponding `IConfiguration` is passed to the agent.
```

Comment:
Similarly, here I'd make clear that this is an `appsettings.json` sample for ASP.NET Core.

Reply:
added